modules/gdnavigation/gd_navigation_server.h

```diff
@@ -80,6 +80,7 @@ class GdNavigationServer : public NavigationServer3D {
 
 	bool active = true;
 	Vector<NavMap *> active_maps;
+	Vector<uint32_t> active_maps_update_id;
```

Comment:
Now that we have `LocalVector` I think it's much better use it, `Vector` is good when you need to pass it around.

(Vector and LocalVector API are the same, you can just change the class.)